### PR TITLE
Add the instruction on how to configure the eventing sources with the operator

### PR DESCRIPTION
**Pull request by houshengbo**
## Proposed Changes 

- This PR adds the instruction on how to configure the eventing sources with the eventing CR.

### CODE REVIEW

File: docs/admin/install/knative-with-operators.md [hunk 1]
```diff
@@ -463,6 +463,166 @@ NAME               VERSION             READY   REASON
 knative-eventing   <version number>    True
 ```
 
+### Installing with Different Eventing Sources
```
[the remark this answers is not in the document]
Comment (snneji) [reply docs/admin/install/knative-with-operators.md:466]: Our style guide says that headings should be in sentence case not title case

**Review by snneji — CHANGES_REQUESTED** [review]
Looks good. Only a couple of tiny typos. 

Just a note, we have recently changed the way we document `kubectl apply` commands (for more info, see https://github.com/knative/docs/issues/3813 and [Documenting YAML](https://dev-knative.netlify.app/development/help/contributor/style-guide/documenting-code/#documenting-yaml) in the Style Guide). 

I am currently working on PRs to update all commands to the new format, so you can leave the commands as they are in this PR. I can update them as part of working on that issue.

**Comment by omerbensaadon** [comment]
/approve 

Created https://github.com/knative/docs/issues/3956 to address aligning to Style Guide

**Pull request merged**